features/ubuntu_pro.feature

```diff
@@ -336,16 +337,69 @@ Feature: Command behaviour when auto-attached in an ubuntu PRO image
         And  I verify that running `systemctl status ua-reboot-cmds.service` `as non-root` exits `0,3`
 
         Then stdout matches regexp:
-            """
-            .*status=0\/SUCCESS.*
-            """
+        """
+        .*status=0\/SUCCESS.*
+        """
 
         Examples: ubuntu release
            | release | fips-s   | cc-eal-s | cis-s    | infra-pkg | apps-pkg | cis_or_usg |
            | xenial  | disabled | disabled | disabled | libkrad0  | jq       | cis        |
            | bionic  | disabled | disabled | disabled | libkrad0  | bundler  | cis        |
            | focal   | disabled | n/a      | disabled | hello     | ant      | usg        |
```

Comment:
Is there no inclusion of jammy in this list?

Reply:
@lucasmoura  would it be better or worse to break the enabling `usg`/`cis` on pro into it's own test that doesn't include jammy? Then the main pro test could be used for all series without duplicating

Reply:
@orndorffgrant just noticed that we also enable the `cis|usg` on the proxy tests. What if we keep it that only on those tests then ?

Reply:
@lucasmoura good point, that works for me!